### DISCUSSION TITLE
It is really build App Bundle with Quasar (cordova) or only APK file for android? Thank you)

### DISCUSSION THREAD

**Discussion by gesha50** [quasarframework/quasar#11655] [CLI - Capacitor / Cordova modes] [answered]
It is really build App Bundle with Quasar (cordova) or only APK file for android? Thank you)

**Comment by metalsadman** [comment]
you need macOs to build an iOs bundle.

**Comment by pikil** [comment, marked as answer]
You can build APK via quasar commands and then convert it to bundle according to this:
https://stackoverflow.com/questions/69357694/unable-to-build-app-bundle-aab-in-cordova
